resources/views/rss.blade.php

```diff
@@ -1,34 +1,34 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<rss version="2.0"
-     xmlns:content="https://purl.org/rss/1.0/modules/content/"
-     xmlns:dc="https://purl.org/dc/elements/1.1/"
-     xmlns:atom="https://www.w3.org/2005/Atom"
->
-    <channel>
-        <title>Tomas Votruba writes about PHP and education</title>
-        <link>https://tomasvotruba.com/</link>
-        <description>{{ \App\Enum\Design::BLOG_TITLE }}</description>
-        <pubDate>{{ date('r') }}</pubDate>
-        <atom:link href="https://tomasvotruba.com/rss.xml" rel="self" type="application/rss+xml"/>
-        <lastBuildDate>{{ $most_recent_post_date_time_stamp }}</lastBuildDate>
+@php
+    /** @var $posts \App\Entity\Post[] */
+@endphp
 
-        @php
-            /** @var $posts \App\Entity\Post[] */
-        @endphp
+<feed xmlns="http://www.w3.org/2005/Atom">
+    <id>{{ action(\App\Http\Controllers\RssController::class) }}</id>
+    <link href="{{ action(\App\Http\Controllers\RssController::class) }}" />
```

Comment:
You should add link to homepage here.

When clicked on title, Feedly now opens feed url instead.

Reply:
Oh, I see. Could you PR it? 🙏